src/main/java/kostka/moviecatalog/controller/RuntimeConfigurationController.java

```diff
@@ -35,7 +35,8 @@ public RuntimeConfiguration create(@RequestBody @Valid final RuntimeConfigDto dt
     public RuntimeConfiguration updateConfiguration(
             @RequestBody final RuntimeConfigDto dto) {
         LOGGER.info("update runtime config '{}' request", dto.getConfigName());
-        return runtimeConfigurationService.update(dto);
+        var validatedRuntimeConfiguration = runtimeConfigurationService.getValidatedRuntimeConfiguration(dto);
+        return runtimeConfigurationService.save(validatedRuntimeConfiguration);
```

Comment:
Možná bych tyhle 2 metody nějak spojil. Ať se ta validace provádí v rámci save. Přece je to jenom public metoda a klient by tam mohl poslat nevalidní objekt.
Ale nezkoumal jsem úplně do detailu, jak to vše spolu funguje. Jinak supr, že to frčí :)

Reply:
Ta anotace kontroluje validní hodnotu co vrací metoda. Když tu metodu ale zavolám v rámci jedné servisy, validace se neprovede... Chtě jsem to udělat jako součást update metody:
Update() {
Getvalidatedruntime... ()
Save()
} 

A tu anotaci použít jen u té get validate metody, ale to. Neprošlo.. Ještě to ale zkusím promyslet :-)

Reply:
trosku jsem to poupravil, muzes to Rade zkontrolovat?

Reply:
To vypadá, že by mohlo fungovat...